app/src/main/java/org/alphatilesapps/alphatiles/Start.java

```diff
@@ -331,7 +338,7 @@ public void buildTilesArray() {
                 Tile tile = new Tile(thisLineArray[0], thisLineArray[1], thisLineArray[2], thisLineArray[3], thisLineArray[4], thisLineArray[5], thisLineArray[6], thisLineArray[7], thisLineArray[8], thisLineArray[9], thisLineArray[10], 0, 0, 0, stageOfFirstAppearance, stageOfFirstAppearanceType2, stageOfFirstAppearanceType3);
                 if (!tile.hasNull()) {
                     tileList.add(tile);
-                    if (!tile.tileType.equals("SAD")) {
+                    if (!tile.tileType.equals("SAD") && !tile.audioForTile.equals("zz_no_audio_needed")) {
```

Comment:
Yes, that is correct. Right now, the default for language packs that don't have tile audio is to put "X" for audio name. zz_no_audio_needed is for pieces of tiles that don't occur on their own, only as components of complex tiles. Currently, we need these "tile pieces" to appear in the word list because we need their type info to know how they combine with other parts of the complex tiles. But they don't have their own audio.